Web Inspector: Console: timestamps take up different amounts of space

dcrousso opened the pull request:
#### 0d3b3290d31fa516c75f370fc63aeec26b8ec5d8
<pre>
Web Inspector: Console: timestamps take up different amounts of space
<a href="https://bugs.webkit.org/show_bug.cgi?id=256590">https://bugs.webkit.org/show_bug.cgi?id=256590</a>

Reviewed by Patrick Angle.

* Source/WebInspectorUI/UserInterface/Views/ConsoleMessageView.css:
(.console-message .timestamp):

Canonical link: <a href="https://commits.webkit.org/264161@main">https://commits.webkit.org/264161@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/932ddb50d25f15d5550045ee993259df1e69630a

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/6102 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/6286 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/6469 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/7665 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/6450 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/6504 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/6240 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/9326 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/6212 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/6230 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/5516 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/7727 "Built successfully") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/39/builds/3706 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/5497 "Passed tests") | [❌ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/13408 "12 flakes 176 failures") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/5571 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/5575 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/7810 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/6048 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/4948 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/5469 "Built successfully") | | | 
| [❌ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/1637 "Failed to push commit to Webkit repository") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/9609 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/5837 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

patrickangle commented on Source/WebInspectorUI/UserInterface/Views/ConsoleMessageView.css:
```diff
@@ -313,6 +313,9 @@
     float: right;
     margin-inline-start: 12px;
+    padding-top: 2px;
+    font-family: Menlo, monospace;
```
<grumbles about the fact we don't use the system monospace font…>

dcrousso replied:
yeah i personally do prefer the non-monospaced look, but not if it shifts around the other stuff :/

i suppose another option is to just calculate the maximum `width` and hardcode that?

dcrousso committed:
Web Inspector: Console: timestamps take up different amounts of space
https://bugs.webkit.org/show_bug.cgi?id=256590

Reviewed by Patrick Angle.

* Source/WebInspectorUI/UserInterface/Views/ConsoleMessageView.css:
(.console-message .timestamp):

Canonical link: https://commits.webkit.org/264161@main
